Assets/Scripts/PlayerController.cs

```diff
@@ -1,5 +1,7 @@
 using System;
+using Unity.Mathematics;
```

Comment:
[It's not recommended to use `Unity.Mathematics`](https://docs.unity3d.com/Packages/com.unity.mathematics@1.3/manual/compatibility.html):

> [!IMPORTANT]
> For performance reasons, if your project uses the Mono compiler, you should continue to use the mathematical operations in `Mathf`, rather than Unity Mathematics.